src/olympia/addons/indexers.py

```diff
@@ -152,19 +152,6 @@ def get_mapping(cls):
                             'raw': {'type': 'keyword'}
                         },
                     },
-                    # TODO: Can be removed once we have `name.raw` indexed
-                    'name_sort': {'type': 'keyword'},
-                    'persona': {
-                        'type': 'object',
-                        'properties': {
-                            'accentcolor': {'type': 'keyword', 'index': False},
-                            'author': {'type': 'keyword', 'index': False},
-                            'header': {'type': 'keyword', 'index': False},
-                            'footer': {'type': 'keyword', 'index': False},
-                            'is_new': {'type': 'boolean', 'index': False},
-                            'textcolor': {'type': 'keyword', 'index': False},
-                        }
-                    },
```

Comment:
I think you removed too much, the issue was just about removing `name_sort`, the `persona` property needs to stay :)

Reply:
Thanks for that heads up! Just gotta figure out what tests are failing for Travis now